src/rust/aggregator/src/data/user_history.rs

```diff
@@ -46,7 +46,10 @@ impl Data for UserHistory {
     }
 
     async fn process_and_save_historical_data(&mut self) -> DataAggregationResult {
-        self.process_and_save_internal().await
+        tracing::info!("[Aggregator] Start processing UserHistory.");
+        let res = self.process_and_save_internal().await;
+        tracing::info!("[Aggregator] Finish processing UserHistory.");
```

Comment:
I think these two logging statements are redundant since there are already start/end log commands in `process_and_save_interal()`

How about taking these out?

Reply:
reverted the file